pkg/gatherers/clusterconfig/gather_node_logs.go

```diff
@@ -50,7 +63,7 @@ func (g *Gatherer) GatherNodeLogs(ctx context.Context) ([]record.Record, []error
 }
 
 func gatherNodeLogs(ctx context.Context, client corev1client.CoreV1Interface) ([]record.Record, []error) {
-	nodes, err := client.Nodes().List(ctx, metav1.ListOptions{LabelSelector: "node-role.kubernetes.io/master"})
+	nodes, err := client.Nodes().List(ctx, metav1.ListOptions{})
```

Comment:
The reason for this was to get only logs from master nodes, I guess. We want to filter everyrthing right?

Reply:
I think this might mean much, much more data than before. Can't we filter just master node logs? I think usually there's more worker nodes than master nodes.

Reply:
This is correct. The task doesn't limit the log gathering by the cluster type.